src/utils/components/WarningWindow.vue

```diff
@@ -0,0 +1,108 @@
+<script setup>
+import { computed, ref, toRefs } from 'vue'
+import { useI18n } from 'vue-i18n'
+import { useStore } from 'vuex'
+
+const props = defineProps({
+    title: {
+        type: String,
+        default: '',
+    },
+    /**
+     * Hide the modal with backdrop, can be used to temporarily hide the modal without loosing its
+     * content
+     */
+    hide: {
+        type: Boolean,
+        default: false,
+    },
+    /** Add a minimize button in header that will hide/show the body */
+    hasMinimize: {
```

Comment:
this variable seems to do nothing

Reply:
good point I'll remove this this is a copy from ErrorWindow